### PR TITLE
py3k megablast

### PR DESCRIPTION
**Pull request by bgruening**
If that passes can I request a review @peterjc :)

### CODE REVIEW

File: tools/megablast_xml_parser/megablast_xml_parser.py [hunk 1]
```diff
@@ -1,78 +1,71 @@
...
+import re
+import sys
+import xml.etree.cElementTree as ElementTree
 
-def stop_err( msg ):
-    sys.stderr.write( "%s\n" % msg )
+
+def stop_err(msg):
+    sys.stderr.write("%s\n" % msg)
     sys.exit()
```
Comment (bernt-matthias) [inline tools/megablast_xml_parser/megablast_xml_parser.py:10]: Just `sys.exit("%s\n" % msg") `

File: tools/megablast_xml_parser/megablast_xml_parser.py [hunk 2]
```diff
@@ -1,78 +1,71 @@
...
+        stop_err("Invalid data format.")
     # turn it into an iterator
-    context = iter( context )
+    context = iter(context)
     # get the root element
     try:
-        event, root = context.next()
-    except:
-        stop_err( "Invalid data format." )
+        event, root = context.__next__()
```
Comment (bernt-matthias) [inline tools/megablast_xml_parser/megablast_xml_parser.py:38]: Is this really a `__` function?

Comment (peterjc) [reply tools/megablast_xml_parser/megablast_xml_parser.py:38]: Why not ``event, root = next(context)`` which is Python 3 style?

File: tools/megablast_xml_parser/megablast_xml_parser.py (hunk 1 above)
Comment (peterjc) [reply tools/megablast_xml_parser/megablast_xml_parser.py:10]: I think ``sys.exit(msg)`` should work just as well:

```
$ python -c "import sys; sys.exit('Bye')"
Bye
```

Comment (nsoranzo) [reply tools/megablast_xml_parser/megablast_xml_parser.py:10]: > I think `sys.exit(msg)` should work just as well:

Yes, and there's no need to define a `stop_err` function to wrap that.

**Review by bernt-matthias — APPROVED** [review]

**Comment by bernt-matthias** [comment]
Anything open here? Or reasons against merging?

**Pull request merged**